samples/caf_sensor_manager/sample.yaml

```diff
@@ -25,7 +25,7 @@ tests:
     integration_platforms:
       - nrf52840dk_nrf52840
       - qemu_cortex_m3
-    extra_args: remote_OVERLAY_CONFIG=sensor_stub_overlay.conf
```

Comment:
What about other uses of `remote_OVERLAY_CONFIG` in the file? Are these OK?

Reply:
I can see only one left (L43) and this is ok - this is multicore build and sensor simulation is placed on remote core.